src/MatrixClient.ts

```diff
@@ -1977,6 +1978,20 @@ export class MatrixClient extends EventEmitter {
         return this.doRequest("GET", url);
     }
 
+    /**
+     * Get relations for a given event.
+     * @param {string} roomId The room ID to for the given event.
+     * @param {string} eventId The event ID to list relations for.
+     * @param {string?} relationType The type of relations (e.g. `m.room.member`) to filter for. Optional.
+     * @param {string?} eventType The type of event to look for (e.g. `m.room.member`). Optional.
+     * @returns {Promise<{chunk: any[]}>} Resolves to an object containing the chunk of relations
+     */
```

Comment:
The comment is apparently copied from elsewhere

```suggestion
    /**
     * Get list of message and state events for a room using /messages.
     * @param {string} roomId The room ID to for the given event.
     * @param {string} dir Thedirection to paginate in.
     * @param {Partials<{filter: string, from: string, limit: number, to: string}>} opts Additional options to set for the request. Can be `filter`, `from`, `limit` and `to`.
     * @returns {Promise<{chunk: GetRoomMessagesResponse}>} Resolves to an object containing the chunk of messages
     */
```